R/utils.R

```diff
@@ -751,7 +751,7 @@ inline_html_styles <- function(html, css_tbl) {
 }
 
 is_equal_to <- function(x, y) {
```

Comment:
1. If I'm reading this correctly, `x` can be a vector with any length and `y` must be a vector of length exactly 1? If so, the name is a bit confusing.
2. `isTRUE` is entirely redundant here, isn't it?
3. If you remove the `isTRUE`, the `function(a, b)` also becomes redundant, i.e. `vapply(x, identical, logical(1), y)`.
4. Couldn't you just replace `is_equal_to(x, y)` with `x == y` (other than NA/NaN handling)?

Reply:
For this, both `x` and `y` can be of any size (in `is_equal_to()`). Comparisons for each value of `x` will be made with the entire vector of `y` (although in our actual use of `is_equal_to()` we bind `0` to `y`). 

I agree that the `isTRUE` and the anonymous function are not required. The `vapply(x, identical, logical(1), y)` is all that's required.

`x == y` would indeed work fine in most cases. In the cases of:

```r
# used currently:
small_pos <-
  ((x >= 1 & x < 10) |
     (x <= -1 & x > -10) |
     is_equal_to(x, 0)) & !is.na(x)
```

and 

```r
# could be used instead:
small_pos <-
  ((x >= 1 & x < 10) |
     (x <= -1 & x > -10) |
     x == 0) & !is.na(x)
```

Both work because any `NA`s produced before the `&` would still result in an overall `FALSE`. 

I believe that this util function was made just in case we wanted to use it elsewhere but that hasn't been the case so far. But I'm in favor of removing it entirely and using `x == 0` in place of `is_equal_to()` for the one time it's called.